src/pathstyle.rs

```diff
@@ -52,6 +52,22 @@ fn coord_string(p: Point) -> String {
     format!("{},{}", round2(p.x), round2(p.y))
 }
 
+struct SvgCoords(String);
```

Comment:
coming into this as a reviewer without any context I would personally benefit from like a line of docs here explaining a bit more about what this is for?

Reply:
Added docs per IM.